src/commands/base-command.mjs

```diff
@@ -8,29 +8,25 @@ import debug from 'debug'
 import merge from 'lodash/merge.js'
 import { NetlifyAPI } from 'netlify'
 
-import { getAgent } from '../lib/http-agent.cjs'
-// TODO: use named imports again once the imported file is esm
-import utils from '../utils/index.cjs'
-
-const {
+import { getAgent } from '../lib/http-agent.mjs'
+import {
   NETLIFY_CYAN,
-  StateConfig,
   USER_AGENT,
   chalk,
   error,
   exit,
-  getGlobalConfig,
   getToken,
-  identify,
   log,
   normalizeConfig,
-  openBrowser,
   padLeft,
   pollForToken,
   sortOptions,
-  track,
   warn,
-} = utils
+} from '../utils/command-helpers.cjs'
+import getGlobalConfig from '../utils/get-global-config.cjs'
+import { openBrowser } from '../utils/open-browser.cjs'
+import StateConfig from '../utils/state-config.mjs'
+import { identify, track } from '../utils/telemetry/index.mjs'
```

Comment:
I started unusing `/utils/index.cjs` :

1. It is easier to migrate
2. It makes node not require everything at once, which might come in handy once the commands are not all loaded always, and only the command in use.